nominees/nomie.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Nomie",
+  "description": "An open source life-tracker app",
+  "website": "https://nomie.app/",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/open-nomie/nomie5/blob/v5-develop/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 3,
+      "evidenceText": "This app has a great privacy policy. It doesn't collect, use, sell or track personal data. Users can store their data locally or encrypted in Blockstack app",
+      "evidenceURL": "https://nomie.app/privacy"
+    }
+  ],
+  "sectors": [
+    "Data Collection",
+    "Health"
+  ],
+  "type": [
+    "software"
+  ],
+  "repositoryURL": "https://github.com/open-nomie/nomie5",
+  "organizations": [
+    {
+      "name": "Nomie",
+      "website": "https://github.com/open-nomie",
```

Comment:
@Rolikasi for the website field under organization, it would be more relevant to indicate their [website](https://nomie.app/) instead of the Github [repo](https://github.com/open-nomie). Thanks.